src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaTransactionRetrieverTests.cs

```diff
@@ -111,15 +112,12 @@ public void GetMempoolTransactionsByPriority_should_return_transactions_in_decre
             for (var i = 0; i < maxCount; i++)
             {
                 retrievedTransactions[i].IsPublicTransaction.Should().Be(expectedTransactions[i].IsPublicTransaction);
-                if (i == 0)
-                {
-                    continue;
-                }
+                if (i == 0) continue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)